readyplayerme/rpm-unity-sdk-avatar-creator
# [SDK-453] Feature/precompile support

## [SDK-453](https://ready-player-me.atlassian.net/browse/SDK-453)

## Description

-  this adds support for using the precompile endpoint to slightly increase avatar update/load times during avatar creation process.







## How to Test

- Import CAC package and run the sample scene. 
- switch categories  and wait  a few seconds (as precompile takes some time to complete
- then switch to one of the first 20 assets of that category
- In theory, these first 20 should generally load faster (0.5s -> 0.8s for outfits) compared to others (~1s -> 2s)



## Checklist








[SDK-453]: https://ready-player-me.atlassian.net/browse/SDK-453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
